app/controllers/foo_controller.rb

```diff
@@ -1,13 +1,13 @@
-class FooController < ActionController::Base  
+class FooController < ActionController::Base
   def new_user
-    user = User.first(:conditions => ("username = '#{params[:username]}'"))
-    user = User.first(:conditions => ("username = '#{params[:username]}'"))
-    user = User.first(:conditions => ("username = '#{params[:username]}'"))
+    user = User.first(conditions: "username = '#{params[:username]}'")
+    user = User.first(conditions: "username = '#{params[:username]}'")
+    user = User.first(conditions: "username = '#{params[:username]}'")
     if user.blank?
       render text: user.username
       return
     end
 
-    render text: "test"
+    render text: 'test'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.